airflow/providers/microsoft/azure/triggers/wasb.py

```diff
@@ -102,26 +102,28 @@ class WasbPrefixSensorTrigger(BaseTrigger):
             ``copy``, ``deleted``
     :param delimiter: filters objects based on the delimiter (for e.g '.csv')
     :param wasb_conn_id: the connection identifier for connecting to Azure WASB
-    :param poke_interval:  polling period in seconds to check for the status
+    :param check_options: Optional keyword arguments that
+        `WasbAsyncHook.check_for_prefix_async()` takes.
     :param public_read: whether an anonymous public read access should be used. Default is False
+    :param poke_interval:  polling period in seconds to check for the status
     """
 
     def __init__(
         self,
         container_name: str,
         prefix: str,
-        include: list[str] | None = None,
-        delimiter: str = "/",
```

Comment:
Would these arguments now need to be passed in under `check_options`? And would this call for this PR to be included in a major release considering it could be a breaking change(assuming we call Triggerers as public interfaces)?

Reply:
> Would these arguments now need to be passed in under check_options? 

 Yes

> And would this call for this PR to be included in a major release considering it could be a breaking change(assuming we call Triggerers as public interfaces)?

Hm.... in this case , maybe this is needed as well 🤔

Reply:
@eladkal @potiuk would like to hear your thoughts on whether we would need to consider this for a major release of the microsoft azure provider.

Reply:
I will skip azure from this wave.
We also have https://github.com/apache/airflow/pull/36622#discussion_r1457115849 which is going to be breaking change as well so we can just mark both as breaking change.

@Lee-W Can you please follow what I wrote in https://github.com/apache/airflow/pull/36622#discussion_r1457115849 ?

Reply:
Sure! Will create a pr for this

Reply:
https://github.com/apache/airflow/pull/36940